src/codegen/values_runtime_proxy.cpp

```diff
@@ -19,6 +19,36 @@ namespace peloton {
 namespace codegen {
 
 //===----------------------------------------------------------------------===//
+// OUTPUT BOOL
+//===----------------------------------------------------------------------===//
+
+// Get the symbol name for ValuesRuntime::OutputBool()
+const std::string &ValuesRuntimeProxy::_OutputBool::GetFunctionName() {
+  static const std::string kOutputTinyIntFnName =
```

Comment:
The name does not matter on how the system executes, but it would be neat if we align this with the function name, since we have been aligning it.